src/Core/Framework/Api/ApiException.php

```diff
@@ -249,6 +251,15 @@ public static function unsupportedOperation(string $operation): self
         );
     }
 
+    public static function unsupportedStoreApiSchemaEndpoint(): self
+    {
+        return new self(
+            Response::HTTP_BAD_REQUEST,
+            self::API_UNSUPPORTED_STORE_API_SCHEMA_ENDPOINT,
+            'The store API does not support the entity schema endpoint. Use /store-api/_info/openapi3.json for the OpenAPI specification. The entity schema endpoint is only available for the admin API.',
```

Comment:
```suggestion
            'The Store-API does not support the entity schema endpoint. Use `/store-api/_info/openapi3.json` for the OpenAPI specification.
```

not sure, if we need to mention the Admin API